lib/signed-xml.js

```diff
@@ -917,7 +923,7 @@ SignedXml.prototype.createSignature = function(prefix) {
     prefix = '';
   }
 
-  var signatureValueXml = "<" + prefix + "SignatureValue>" + this.signatureValue + "</" + prefix + "SignatureValue>"
+  var signatureValueXml = "<" + prefix + "SignatureValue>" + this.signatureValue.toString('base64') + "</" + prefix + "SignatureValue>"
```

Comment:
Is it possible for `this.signatureValue` to be undefined or null which would cause `toString` to fail?

Reply:
Similar to above. `createSignature` is an internal method of `computeSignature`. Note that the documentation is just the README for "public" methods but those are the ones meant to be used by consumers to have proper usage of the library.